plugins/test/unit/__init__.py

```diff
@@ -9,6 +10,10 @@
 devel_base.block_load_conf()
 
 
+if django.VERSION >= (1, 9):
+    django.setup()
```

Comment:
Good question. I'm using the "run-tests.py" script. We don't have a ``manage.py`` in pulp 2.